examples/StandardFirmata/StandardFirmata.ino

```diff
@@ -20,13 +20,20 @@
 
   See file LICENSE.txt for further informations on licensing terms.
 
-  Last updated by Jeff Hoefs: August 9th, 2015
+  Last updated by Jeff Hoefs: October 3rd, 2015
 */
 
 #include <Servo.h>
 #include <Wire.h>
+#include <ArduinoUnit.h>
 #include <Firmata.h>
 
+// SoftwareSerial is only supported for AVR-based boards
+#if defined(ARDUINO_ARCH_AVR)
```

Comment:
This will fail in Arduino 1.0.x since the constant `ARDUINO_ARCH_AVR` was not added until Arduino 1.5.x. Therefore SoftwareSerial will not work with Firmata in Arduino 1.0.x as this is written.